src/app/machines/components/MachineForms/MachineActionFormWrapper/MachineActionFormWrapper.tsx

```diff
@@ -18,19 +20,26 @@ import SetZoneForm from "app/base/components/node/SetZoneForm";
 import TestForm from "app/base/components/node/TestForm";
 import type { HardwareType } from "app/base/enum";
 import { useScrollOnRender } from "app/base/hooks";
-import type { ClearSidePanelContent, SetSearchFilter } from "app/base/types";
+import type {
+  APIError,
+  ActionStatuses,
+  ClearSidePanelContent,
+  SetSearchFilter,
+} from "app/base/types";
```

Comment:
instead of importing these individual types I'd suggest to use `ReturnType` TypeScript util. Will make it easier to maintain.